README.md

```diff
@@ -4,38 +4,80 @@ Dummy Credentials repository to demonstrate how to bootstrap a tower instance wi
 
 ## Getting Started
 
-This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project. As this project contains sensitive information, it should only be used locally.
+This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project.
 
-## Usage
+## Usage 
 
-The example below runs the `bootstrap.yml` playbook using this project as the inventory source, where `--ask-vault-pass` is the password used to access the encrypted variables.
+The projects `SAMPLE_CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own environment. Once these variables have been changed, the `bootstrap.yml` playbook is used to consume this configuration file,encrypt any variables containing sensitive information using the supplied Ansible vault password, and then place each of the variables into the relevant group_vars file.
+
+## Setup
+
+**NOTE**: This project should be forked and made private before proceeding with the following setup steps.
+
+1. Clone this project
 
 ```bash
-ansible-playbook -i <path-to-tower-dummy-credentials-project> playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment> --ask-vault-pass
+git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-## Ansible-Vault
+2. Copy and rename the `SAMPLE_CREDENTIAL_CONFIG.yml` file locally. As the local copy of the config will sensitive information in plaintext, it should only be stored and referenced locally, and caution taken to not check it into a public repository.
 
-Credentials stored in this project are encrypted using [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html). Ansible vault supports the encryption of entire [files](https://docs.ansible.com/ansible/latest/user_guide/vault.html#creating-encrypted-files), and also individual [string values](https://docs.ansible.com/ansible/latest/user_guide/vault.html#use-encrypt-string-to-create-encrypted-variables-to-embed-in-yaml) that can embedded within YAML files. The default encrypted string value of all variables within this project along with the password to decrypt these variables is `CHANGEME`.
+```bash
+cp SAMPLE_CREDENTIAL_CONFIG.yml local_credentials_config.yml
+```
 
-### Files
+3. Change the variable values in the newly created local copy of the `SAMPLE_CREDENTIAL_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
```

Comment:
I think this could be re-worded as "Change the variable values in the newly created local copy of the local_credentials_config.yml file" to reflect changes in the previous step? Rather than SAMPLE_CREDENTIAL_CONFIG.yml